CompatibilityRequirements.md

```diff
@@ -58,7 +58,7 @@ use of the previous version of the specification. However if it is
 considered that there will be no or few such implementations, or if
 they would always be provided as an add-on by the developers of vendor
 implementations, then this requirement may be relaxed after careful
-consideration by the specification's expert group.
+consideration by the specification project team.
```

Comment:
```suggestion
consideration by the specification project.
```

In most cases, you changed "specification's expert group" to "specification project". You could remove the "team" here too, for consistency.

Reply:
Thanks for the review!  I fixed this and all your other comments.